en.json

```diff
@@ -9,7 +9,7 @@
 			"option-spellcheck": "Spell check",
 			"option-spellcheck-description": "Turns on the spell checker. Currently only works for English (US).",
 			"spellcheck-dict": "Spellcheck dictionary",
-			"spellcheck-dict-empty": "The spellcheck dictionary is empty.",
+			"spellcheck-dict-emptåy": "The spellcheck dictionary is empty.",
```

Comment:
Please do not translate `en.json`, thanks!

Reply:
Ok , that was a typo thou... thanks anyways.. :)

Reply:
Ah okay. Once you have the time to change it back I can merge it, thanks!